client/ayon_core/hosts/houdini/plugins/publish/collect_cache_farm.py

```diff
@@ -7,7 +7,7 @@
 class CollectDataforCache(pyblish.api.InstancePlugin):
     """Collect data for caching to Deadline."""
 
-    order = pyblish.api.CollectorOrder + 0.04
+    order = pyblish.api.CollectorOrder + 0.11
```

Comment:
Would it help if we add a comment?
```suggestion
    # Run after Collect Frames
    order = pyblish.api.CollectorOrder + 0.11
```